odl/solvers/functional/default_functionals.py

```diff
@@ -223,6 +229,8 @@ def __init__(self, vfspace, exponent=None):
             0 and 1 are currently not supported due to numerical
             instability. Infinity gives the supremum norm.
             Default: ``vfspace.exponent``, usually 2.
+        range : `Field`, optional
+            Range of the functional. Default: `vfspace.field`.
```

Comment:
For this kind of thing you need to use double backticks. 
With single backticks, Sphinx will try to generate a hyperlink, which fails because Sphinx doesn't know the type of `vfspace`. Single backticks are usable for class names and glossary terms mostly, other things that are not referable need to go in double backticks.